ui/service.go

```diff
@@ -65,17 +68,24 @@ func New(things mainflux.ThingsServiceClient, sdk sdk.SDK) Service {
 	}
 }
 
-func (gs *uiService) Index(ctx context.Context, token string) ([]byte, error) {
-	tpl := template.New("index")
+func ParseTemplate(tmp string) (*template.Template, error) {
+	tpl := template.New(tmp)
```

Comment:
Do we need this at all, when we create a `tpl` by calling `ParseFiles()`?

Reply:
It must be called before the template is parsed.